pkg/dbrole/dbrole_test.go

```diff
@@ -49,6 +49,8 @@ func TestIsDbRoleTenantScoped(t *testing.T) {
 	assert.False(dbrole.NO_ROLE.IsDbRoleTenantScoped())
 	assert.False(dbrole.READER.IsDbRoleTenantScoped())
 	assert.False(dbrole.WRITER.IsDbRoleTenantScoped())
+	assert.True(dbrole.TENANT_INSTANCE_READER.IsDbRoleTenantScoped())
```

Comment:
You should add assertions for INSTANCE_READER & INSTANCE_WRITER roles

Reply:
will add

Reply:
Done